modules/super-admin/components/product-listing/ProductListingNavbar.tsx

```diff
@@ -1,15 +1,45 @@
 import { ArrowUp } from 'iconsax-react';
 import Link from 'next/link';
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { ProductStatistics } from '../../../../@types';
 
 const ProductsListingNavbar = () => {
+  const [productStats, setProductStats] = useState<ProductStatistics | null>(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const authToken =
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImE3ODlmMzVlLTAwNjQtNDM1Yy1hMTI5LTI3OGY5YzFhYTFkYiIsImlhdCI6MTY5NzE0MjQwMX0.Np4b4179X0XfbmBlN9ev_EYc8DLPyxtW7UOcf3b4dxI';
+        const response = await axios.get<ProductStatistics>('https://staging.zuri.team/api/admin/product/all', {
+          headers: {
```

Comment:
Please create an env and store this https://staging.zuri.team/api as the  baseURL